Shopify/cli
# Fix extension template IDs

### WHY are these changes introduced?

In https://github.com/Shopify/cli/pull/1941 I added some external IDs that don't match with the current ones defined here: https://shopify.dev/docs/apps/tools/cli/commands#supported-extension-types

### WHAT is this pull request doing?

Keep the current external IDs for templates

### How to test your changes?

`p shopify app generate extension --type wrong` and check the list of valid types

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
